src/bundle/Resources/public/js/scripts/admin.notifications.js

```diff
@@ -6,10 +6,10 @@
         const { onShow, label, message, rawPlaceholdersMap = {} } = detail;
         // @TODO: Unify 'error' and 'danger' label in 3.0.
         const configKey = label === 'danger' ? 'error' : label;
-        const config = eZ.adminUiConfig.notifications[configKey];
+        const config = eZ.adminUiConfig ? eZ.adminUiConfig.notifications[configKey] : null;
```

Comment:
I don't think this is a good approach. This timeout for notifications should be respected.

Reply:
AdminUI is not the only place when we need this. The Password Reset view is in `Security` scope. Should I implement the `adminUiConfig` in `src/bundle/Resources/views/Security/base.html.twig` instead?

Reply:
Good question. My concern is that those notifications will not respect the timeouts. Maybe we should introduce some limited adminUiConfig? Only with needed things?
@Nattfarinn @webhdx